pybossa/cache/site_stats.py

```diff
@@ -30,6 +30,24 @@
 
 session = db.slave_session
 
+def allow_all_time(func):
+    @wraps(func)
+    def wrapper(days=30):
+        if days == 'all':
+            days = 999999
+        return func(days=days)
+
+    return wrapper
+
+def allow_all_time_interval(func):
```

Comment:
Looks to me this function can be removed. We will just need the existing function ```allow_all_time```

Reply:
fixed via https://github.com/bloomberg/pybossa/pull/824/commits/e60184671e6e21cc77396fa7e9e9460e2cc78b6c